frontend/src/components/BCHeader/index.tsx

```diff
@@ -13,89 +13,137 @@ import {
   HeaderMenuItem,
   SideNav,
   SideNavItems,
-  HeaderSideNavItems,
-} from '@carbon/react';
-import { NavLink , Link} from "react-router-dom";
-import * as Icons from '@carbon/icons-react';
-import './BCHeader.scss'
+  HeaderSideNavItems
+} from "@carbon/react";
+import { BrowserRouter, NavLink, Link } from "react-router-dom";
+import * as Icons from "@carbon/icons-react";
+import "./BCHeader.scss";
 import { HeaderContainerProps } from "./definitions";
 
 const BCHeader: React.FC = () => {
   //can only be imported at component level
   const { theme, setTheme } = useThemePreference();
 
-    return (
-      <HeaderContainer
-        render={({ isSideNavExpanded, onClickSideNavExpand }: HeaderContainerProps) => (
-          <Header aria-label="React TS Carbon QuickStart"
+  return (
+    <HeaderContainer
+      render={({
+        isSideNavExpanded,
+        onClickSideNavExpand
+      }: HeaderContainerProps) => (
+        <BrowserRouter>
+          <Header
+            aria-label="React TS Carbon QuickStart"
             className="spar-header"
-            data-testid="bc-header__header">
+            data-testid="bc-header__header"
+          >
             <SkipToContent />
             <HeaderMenuButton
-              aria-label={isSideNavExpanded ? 'Close menu' : 'Open menu'}
+              aria-label={
+                isSideNavExpanded ? "Close menu" : "Open menu"
+              }
               onClick={onClickSideNavExpand}
               isActive={isSideNavExpanded}
             />
-            <Link to="/" className="header-link" data-testid="header-name">
+            <Link
+              to="/"
+              className="header-link"
+              data-testid="header-name"
+            >
               SILVA
             </Link>
             <HeaderNavigation aria-label="BC-Gov Starter">
```

Comment:
Most of these links down here are just for tests. We can prune them as part of this change as well.